packages/wallets/src/features/cfd/flows/OtherCFDs/Dxtrade/AvailableDxtradeAccountsList/__test__/AvailableDxtradeAccountsList.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { PropsWithChildren } from 'react';
+import { APIProvider } from '@deriv/api-v2';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import WalletsAuthProvider from '../../../../../../../AuthProvider';
+import { ModalProvider } from '../../../../../../../components/ModalProvider';
+import AvailableDxtradeAccountsList from '../AvailableDxtradeAccountsList';
+
+const mockShow = jest.fn();
+jest.mock('../../../../../../../components/ModalProvider', () => ({
+    ...jest.requireActual('../../../../../../../components/ModalProvider'),
+    useModal: jest.fn(() => ({
+        ...jest.requireActual('../../../../../../../components/ModalProvider').useModal(),
+        show: mockShow,
+    })),
+}));
+
+const wrapper = ({ children }: PropsWithChildren) => (
+    <APIProvider>
+        <WalletsAuthProvider>
+            <ModalProvider>{children}</ModalProvider>
+        </WalletsAuthProvider>
+    </APIProvider>
+);
+
+describe('AvailableDxtradeAccountsList', () => {
+    it('render component', () => {
+        render(<AvailableDxtradeAccountsList />, { wrapper });
+        expect(screen.getByTestId('dt_icon-dxtrade')).toBeInTheDocument();
+        expect(screen.getByText('Deriv X')).toBeInTheDocument();
+    });
+
+    it('show modal popup upon clicking on the component', () => {
```

Comment:
Minor improvement to the Act statement, to be more specific.
```suggestion
    it('shows DxtradeEnterPasswordModal upon clicking on the TradingAccountCard component', () => {
```